Jenkinsfile

```diff
@@ -3,9 +3,16 @@
 def majorVersion
 def minorVersion
 def patchVersion
+def dockerImage
 
 pipeline {
     agent any
+    options {
+        skipStagesAfterUnstable()
```

Comment:
Just curious what these do? Could you add a comment?

Reply:
Normally, if tests fail, the build should be marked UNSTABLE, not FAILED. Then this option prevents the pipeline to continue other stages.

https://jenkins.io/doc/book/pipeline/syntax/